dev/build/windows/makecoq_mingw.sh

```diff
@@ -1211,6 +1214,10 @@ function make_coq {
     # 2.) clean of test suites fails with "cannot run complexity tests (no bogomips found)"
     # make clean
 
+    # Copy these files somewhere the plugin builds can find them
+    cp dev/ci/ci-basic-overlay.sh /build/
+    cp -r dev/ci/user-overlays /build/
```

Comment:
In #7928, @MSoegtropIMC asked where:

```
. /build/ci-basic-overlay.sh
for overlay in /build/user-overlays/*.sh; do
```

come from. This is where. Note that I just imitated what you had done previously with the `dev/ci/plugins_version/` directory.

> Or is the /build folder something else which exists on the CI machines?

No, it's really the `/build` folder as you created it, nothing else.